test/integration/smart_proxy_test.rb

```diff
@@ -20,14 +20,17 @@ class SmartProxyIntegrationTest < ActionDispatch::IntegrationTest
   end
 
   test "edit page" do
-    visit smart_proxies_path
-    click_link "DHCP Proxy"
-    click_link "Edit"
+    SETTINGS[:organizations_enabled] = false
```

Comment:
Not sure what's causing this yet, but when we edit this fixture, it 404s after submitting because some taxonomies could not be found :disappointed: